code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -144,6 +149,7 @@
 /obj/item/gun_upgrade/trigger/dnalock
 	name = "Frozen Star \"DNA lock\" Trigger"
 	desc = "There are many guns, but that one will be yours. Prevents others from using weapon with this trigger."
+	matter = list(MATERIAL_STEEL = 2, MATERIAL_DIAMOND = 1)
```

Comment:
same as danger zone, doesn't make sense for it to cost diamond

Reply:
DNA locking tech in regular TG costs diamond, so I added diamond for familiarities'